File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/model/MyListener.java
package com.c_store.zhiyazhang.cstoremanagement.model;

/**
 * Created by zhiya.zhang
 * on 2017/5/10 15:02.
 */

public interface MyListener {
    void contractSuccess();
    void contractSuccess(Object object);
    void contractFailed(String errorMessage);
}

File: KotlinCStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/model/MyListener.kt
package com.cstore.zhiyazhang.cstoremanagement.model

/**
 * Created by zhiya.zhang
 * on 2017/5/10 15:02.
 */

interface MyListener {
    fun contractSuccess()
    fun contractSuccess(`object`: Any)
    fun contractFailed(errorMessage: String)
}
